### PR TITLE
Task#4 - Interfaces and dependency management

### PR DESCRIPTION
**Pull request by Jagrmi-C**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,4 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
+vendor/
```
Comment (wshaman) [inline .gitignore:15]: Oh, thank you.

File: courses/task_five/main.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+// People - slice of people
+type People []Person
+
+func (p People) Len() int {
+	return len(p)
+}
+
+func (p People) Less(i, j int) bool {
+	if p[i].birthDay.Equal(p[j].birthDay) {
+		return p[i].firstName < p[j].firstName
```
Comment (wshaman) [inline courses/task_five/main.go:34]: Any `lastName` using?

**Review by wshaman — APPROVED** [review]
Looks good. Just a note on lastName in sort

File: courses/task_five/main.go (hunk 2 above)
Comment (Jagrmi-C) [reply courses/task_five/main.go:34]: update

**Review by VolhaVetrava — APPROVED** [review]

**Pull request merged**